src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/OrganisationalInternalControllerProviderTest.java

```diff
@@ -177,6 +189,21 @@ public void setUpOrganisationForMfaUpdate() {
         when(organisationRepository.findByOrganisationIdentifier(anyString())).thenReturn(getOrgWithMfaStatus());
     }
 
+    @State("Update an Organisation's PBA accounts")
+    public void setUpOrganisationForUpdatingPBAs() {
+        Organisation organisation = new Organisation("Org-Name", OrganisationStatus.ACTIVE, "sra-id",
+                "companyN", false, "www.org.com");
+
+        PaymentAccount paymentAccount = new PaymentAccount("PBA1234567");
+        paymentAccount.setOrganisation(organisation);
+
+        doNothing().when(organisationIdentifierValidatorImplMock).validateOrganisationIsActive(any());
+        when(organisationRepository.findByOrganisationIdentifier(anyString())).thenReturn(organisation);
+        when(paymentAccountRepository.findByPbaNumberIn(anySet())).thenReturn(asList(paymentAccount));
+        when(paymentAccountRepository.saveAll(anyList())).thenReturn(asList(paymentAccount));
```

Comment:
Please use `List.of()` to return a list instead of `Arrays.asList()`

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/b17549eb7a6a39bc04a0482a299739fe4dc1e212